plugin-locationcomponent/src/main/java/com/mapbox/maps/plugin/locationcomponent/LocationComponentExt.kt

```diff
@@ -10,4 +11,44 @@ import com.mapbox.maps.plugin.delegates.MapPluginProviderDelegate
  */
 val MapPluginProviderDelegate.location: LocationComponentPlugin
   @JvmName("getLocationComponent")
-  get() = this.getPlugin(Plugin.MAPBOX_LOCATION_COMPONENT_PLUGIN_ID)!!
\ No newline at end of file
+  get() = this.getPlugin(Plugin.MAPBOX_LOCATION_COMPONENT_PLUGIN_ID)!!
+
+/**
+ * Adds a listener that gets invoked when indicator accuracy radius changes.
+ *
+ * @param listener Listener that gets invoked when indicator accuracy radius changes
+ */
+fun LocationComponentPlugin.addOnIndicatorAccuracyRadiusChangedListener(listener: OnIndicatorAccuracyRadiusChangedListener) {
+  (this as LocationComponentPluginImpl).addOnIndicatorAccuracyRadiusChangedListener(listener)
```

Comment:
We could introduce separate interface for the `addOnIndicatorAccuracyRadiusChangedListener` / `removeOnIndicatorAccuracyRadiusChangedListener` method, thus users will be able to implement it as well and don't depent on our LocationComponentPluginImpl, described the idea at https://github.com/mapbox/mapbox-maps-android-internal/issues/857#issuecomment-1010950807

Reply:
Whit the extension functions, users can invoke them like this `mapView.location.addOnIndicatorAccuracyRadiusChangedListener()`
If move these functions into a new interface, how can users invoke them?

Reply:
I mean we can keep the extension function - and at the same time add the interface with `addOnIndicatorAccuracyRadiusChangedListener` / `removeOnIndicatorAccuracyRadiusChangedListener`, and let `LocationComponentPluginImpl` implement this interface. So, this extension function becomes
```
fun LocationComponentPlugin.addOnIndicatorAccuracyRadiusChangedListener(listener: OnIndicatorAccuracyRadiusChangedListener) {
  (this as NewCoolAccuracyInterface).addOnIndicatorAccuracyRadiusChangedListener(listener)
```
This will allow user to implement it as well, if it's needed.

Reply:
Yeah, that make sense.